Lib/test/test_csv.py

```diff
@@ -1189,6 +1189,8 @@ class TestSniffer(unittest.TestCase):
 ghijkl\0mno
 ghi\0jkl
 """
+    sample15 = 'Timestamp,URL,Title\r\n2020-10-01 17:17:37+08:00,https://www.mozilla.org/en-US/firefox/welcome/2/,"Pocket - Save news, videos, stories and more"\r\n'
```

Comment:
This is a really long line in the test file. I think you can reduce it - the real important thing is just the `\r\n`.